client/my-sites/plans/current-plan/trials/migration-trial-included.tsx

```diff
@@ -0,0 +1,108 @@
+import { localize, translate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import advancedDesignTools from 'calypso/assets/images/plans/wpcom/migration-trial/advanced-design-tools.svg';
+import beautifulThemes from 'calypso/assets/images/plans/wpcom/migration-trial/beautiful-themes.svg';
+import bestInClassHosting from 'calypso/assets/images/plans/wpcom/migration-trial/best-in-class-hosting.svg';
+import googleAnalytics from 'calypso/assets/images/plans/wpcom/migration-trial/google-analytics.svg';
+import jetpackBackupsAndRestores from 'calypso/assets/images/plans/wpcom/migration-trial/jetpack-backups-and-restores.svg';
+import newsletters from 'calypso/assets/images/plans/wpcom/migration-trial/newsletters.svg';
+import seoTools from 'calypso/assets/images/plans/wpcom/migration-trial/seo-tools.svg';
+import spamProtection from 'calypso/assets/images/plans/wpcom/migration-trial/spam-protection.svg';
+import FeatureIncludedCard from '../feature-included-card';
+
+interface Props {
+	translate: typeof translate;
+	displayAll: boolean;
+}
+const MigrationTrialIncluded: FunctionComponent< Props > = ( props ) => {
```

Comment:
Maybe for this section it's better to make it as a `BusinessTrialIncluded` as well, what do you think?